src/AppCommon/ServiceControl/ServiceControlAuditsByBinarySearch.cs

```diff
@@ -1,11 +1,12 @@
-﻿namespace Particular.EndpointThroughputCounter.ServiceControl
+﻿#nullable enable
```

Comment:
* Given https://github.com/Particular/Particular.EndpointThroughputCounter/issues/262 we're past February 2024, this collection method should probably be removed and 4.29.0 made the minimum, but the changes here are slight enough that it's probably OK to just go with it in this PR.